applications/ConvectionDiffusionApplication/tests/thermo_mechanical_tests/thermo_mech_accessor/coupled_accessor_test_parameters.json

```diff
@@ -0,0 +1,191 @@
+  {
+    "problem_data"     : {
+        "problem_name"  : "coupled_problem",
+        "parallel_type" : "OpenMP",
+        "start_time"    : 0.0,
+        "end_time"      : 50,
```

Comment:
I think that running a single time step is sufficient in this case.

Reply:
well... the heat flow takes some time to evolve. I'll reduce the simulation time anyways